daemon/src/setup_contract.rs

```diff
@@ -379,24 +380,6 @@ pub async fn roll_over(
         .try_into_msg0()
         .context("Failed to read Msg0")?;
 
-    let complete_fee = match rollover_params.version {
-        RolloverVersion::V1 => {
-            // Note there is actually a bug here, but we have to keep this as is to reach agreement
-            // on the fee for the protocol V1 version.
-            //
-            // The current fee is supposed to be added here, but we never noticed because in V1 the
-            // fee is always charged for one hour using a static rate. This results in applying the
-            // fee in the DLC only for the next rollover (because we do apply the fee in the Cfd
-            // when loading the rollover event). Effectively this means, that we always charged one
-            // rollover too little.
-            rollover_params.fee_account.settle()
```

Comment:
Are we certain that we're not handling rollovers from `RolloverVersion::V1`? Otherwise removing this means that they will eventually fail to rollover.

Reply:
14% of takers are on version 0.4.7 or prior according to our metrics 😨

Reply:
Probably all with no position 🤷‍♂️
I'm happy to keep it here though. Not much harm done.

Reply:
We can certainly make the call that we no longer want to support those takers. But we should probably do that in a separate commit.

Reply:
> We can certainly make the call that we no longer want to support those takers. But we should probably do that in a separate commit.

Let's leave it as it, i.e. with `RolloverVersion::V1`, then the version are compatible.